### PR TITLE
ENH: added schema and exemplar for presentations collection

### PR DESCRIPTION
**Pull request by sbillinge**
I want a presentation collection for our awesome group talks etc.

### CODE REVIEW

**Comment by scopatz** [comment]
Thanks @sbillinge! The code looks good to me.  It is not clear why the tests are failing to build the validator. @CJ-Wright, ideas?

File: regolith/schemas.py [hunk 1]
```diff
@@ -786,6 +803,72 @@
         'title': {'description': 'for example, Dr., etc.',
                   'required': False,
                   'type': 'string'}},
+
```
Comment (CJ-Wright) [inline regolith/schemas.py]: I think this newline is causing most of the problems.

File: regolith/schemas.py [hunk 2]
```diff
@@ -786,6 +803,72 @@
...
+                                   'presentation is associated with.  Should be'
+                                   'discoverable in projects collection',
+                    'required': False,
+                    'type': 'string'},
+        'title': {'description': 'title of the presentation',
+                  'required': True,
+                  'type': 'string'},
+        'type': {'description': 'type of presentation',
+                 'items': {'type': 'string',
+                           'enum': ['award','colloquium', 'contributed_oral',
```
[the remark this answers is not in the document]
Comment (sbillinge) [reply regolith/schemas.py]: ah, that would explain why I couldn't find `enum` when I searched for it in the schema file....I was worried that I was the first person to require an enum and I was feeling a bit bad about it....

[the remark this answers is not in the document]
Comment (sbillinge) [reply regolith/schemas.py]: mmm....come to think of it that other eallowed that chris points out is mine too......better keep my head low.

**Comment by sbillinge** [comment]
dang, still failing tests.  I think there is a missing comma/unmatched bracket in there somewhere but I have to run for a train right now.  Will look later.

**Comment by sbillinge** [comment]
@scopatz @CJ-Wright the code is running the validator tests now, but generating errors in the validation.  Some of these appear to not be related to the code I inserted though.  Do we have to fix so that all the validation tests pass?  I'm happy to do it (I like tests to pass) but just wanted to check there is not something I am missing here.

File: regolith/schemas.py [hunk 3]
```diff
@@ -786,6 +803,73 @@
...
+                       'type': 'string'},
+        'end_year': {'description': 'year the conference or trip ends',
+                     'required': False,
+                     'type': 'integer'},
+        'end_month': {'required': False,
+                      'type': 'string'},
+        'end_day': {'required': False,
+                    'type': 'integer'},
+    },
+    'institution': {'description': 'institution where the'
```
Comment (CJ-Wright) [inline regolith/schemas.py]: Is this part of the presentation? If so it seems that the line above has an extra close `}` which is pushing institution into it's own collection.

**Comment by CJ-Wright** [comment]
@sbillinge can you rebase onto master?

**Comment by sbillinge** [comment]
haha, good question!

**Comment by CJ-Wright** [comment]
It may be possible to pull these changes into yours? (I'm not certain).

**Comment by CJ-Wright** [comment]
I think I was able to fix this on my local. I rebased onto upstream master. @sbillinge is it ok if I open a new PR?

**Comment by sbillinge** [comment]
either is fine.  Happy to merge into my fork.

**Comment by CJ-Wright** [comment]
@sbillinge I think that might be a good course

**Comment by CJ-Wright** [comment]
@scopatz I think this is ready for another round (this also fixes the double schema docs issue).

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**